packages/vite/src/node/plugins/resolve.ts

```diff
@@ -120,6 +122,9 @@ export function resolvePlugin(baseOptions: InternalResolveOptions): Plugin {
         ...baseOptions,
         isFromTsImporter: isTsRequest(importer ?? '')
       }
+      if (!options.isBuild) {
+        options.cjsInclude = undefined
```

Comment:
I don't remember why `cjsInclude` is disabled in dev mode, but it would be nice to figure out why and add a comment here.. :)

Reply:
I suppose it's because `@rollup/plugin-commonjs` isn't used in dev mode 😆